plugins/package-managers/sbt/src/main/kotlin/Sbt.kt

```diff
@@ -182,7 +180,11 @@ class Sbt(
             targetDir.walk().maxDepth(1).filterTo(pomFiles) { it.extension == "pom" }
         }
 
-        return pomFiles.distinct().map { moveGeneratedPom(it) }
+        return pomFiles.distinct().map { pomFile ->
+            moveGeneratedPom(pomFile).onFailure {
+                logger.error { "Moving '$pomFile' failed: ${it.message}" }
```

Comment:
Does this new message print the absolute paths of both the source and target files?

Reply:
Yes, I've just verified this. Actually, the paths are even absolute without the explicit conversion, so I've removed it in a new commit.